lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -296,6 +296,10 @@ defmodule ExUnit.Case do
   @doc false
   def __on_definition__(%{module: mod, file: file, line: line}, name, tags) do
     moduletag = Module.get_attribute(mod, :moduletag)
+    test_attributes =
+      Module.get_attribute(mod, :ex_unit_test_attributes)
+      |> List.wrap()
```

Comment:
I think it should not be needed, however this returns `nil` for `FunctionClashFail2` definition.